README.md

```diff
@@ -9,12 +9,14 @@
 <a href="https://canary-checker.docs.flanksource.com"> <img src="https://img.shields.io/badge/☰-Docs-lightgrey.svg"/> </a>
 </p>
 
+## Documentation
+
+Canary Checker is a multi-tenant synthetic monitoring system native to Kubernetes.  To learn more, please see the [official documentation](https://canary-checker.docs.flanksource.com).
+
 
 ---
 <!--ts-->
-  * [Features](#features)
```

Comment:
I think the Features/Comparisons should be included or duplicated here as the README is often the landing page and we need to "sell" users on why the would even want to go deeper into the docs.